build/build_deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# AppSRE team CD
+
+set -exv
+
+CURRENT_DIR=$(dirname "$0")
+
+BASE_IMG="tbd"
```

Comment:
IMO you could name the base image after the repo: `managed-cluster-validating-webhooks`